content/en/docs/components/pipelines/v2/installation/quickstart.md

```diff
@@ -1,353 +0,0 @@
-+++
```

Comment:
I think we should keep the `v2/installation/quickstart` content instead of redirecting people to v1. It contains practical guidance to actually getting started with the product.